packages/terra-functional-testing/tests/jest/commands/utils/ScreenshotRequestor.test.js

```diff
@@ -142,6 +149,160 @@ describe('ScreenshotRequestor', () => {
     });
   });
 
+  describe('downloadScreenshots', () => {
+    it('downloads the screenshots', async () => {
+      const mockPipe = jest.fn();
+      fetch.mockResolvedValueOnce({
+        ok: true,
+        status: 200,
+        body: {
+          pipe: mockPipe,
+        }
+      });
+      
+      extract.mockResolvedValueOnce();
+
+      const mockCreateWriteStream = jest.fn();
+      const mockRemoveSync = jest.fn();
+      const mockOnFinish = jest.fn();
```

Comment:
These constants are redundant. Since we're already calling `jest.mock('fs-extra');` , the library will already have empty mock implementations. For the expect statements, we can simply use the original methods e.g. `expect(fs.removeSync)`.

Reply:
Fixed here https://github.com/cerner/terra-toolkit/pull/808/commits/8e94641850bdce7c1c53eca331d2198f71d1e3b5#diff-d5e5e839b475dfb26a79344066ab8d867df7550d65a2eed690e281a9d9a0946cR197
It easy to forget such things 😮‍💨